db-migrations/2_test2.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS game_logs (
```

Comment:
This has the game results summary, rather than the (detailed) logs -- do you want to choose a different name for this table?